src/cli/cli_app.js

```diff
@@ -431,7 +434,7 @@ class CLI {
         enigmaContractAddress: this._enigmaContractAddress,
         accountAddress: this._ethereumAddress,
         enigmaContractAbi,
-        accountKey
+        minConfirmations: this._minConfirmations
```

Comment:
By mistake. Fixing.

Reply:
This happened because when I made changes to this file vscode auto-formatted the code and it made the git diff huge because of spaces.  
I manually staged only the line I changed. Probably missed this line. :sob:

Reply:
This is why we need prettier :tada: 

![Screenshot from 2019-11-13 10-05-15](https://user-images.githubusercontent.com/22766533/68744352-1b3bf380-05fd-11ea-8375-356e4ee1bb2f.png)